v2/src/components/question/index.tsx

```diff
@@ -1,5 +1,7 @@
 import React, { Children, PropsWithChildren, useState } from "react";
 
+let styles = require("./question.module.css").default;
```

Comment:
Cant we just import?

Reply:
No, using `import` isn't working on this file as it isn't declared as a module